src/blocks/block-column/components/edit.js

```diff
@@ -172,7 +173,6 @@ class Edit extends Component {
 								>
 									{ __( 'Return to Column Selection', 'atomic-blocks' ) }
 								</Button>
```

Comment:
All the stuff after <Button> to and including </Button> is indented wrong. Not sure why this is passing CS checks.